src/App.js

```diff
@@ -9,7 +10,13 @@ const article = {
 };
 
 const App = () => (
-  <h1>React Article</h1>
+  <div>
+    <Article
+      title={article.title}
+      text={article.text}
+      date={article.date}
```

Comment:
```suggestion
       { ...article }
```
You can use spread operator here to pass the whole object to the props